### PR TITLE
Rosenthal: Add waybar with experimental features

### PR DESCRIPTION
**Pull request by threddast**
Waybar built with the experimental flag has support for additional features, like displaying the number of active workspaces in Hyprland.

### CODE REVIEW

**Comment by rakino** [comment]
I think it's better to make the contribution directly to Guix, how about looking through [Submitting Patches](https://guix.gnu.org/en/manual/devel/en/guix.html#Submitting-Patches) and trying to do so?

File: rosenthal/packages/wm.scm [hunk 1]
```diff
@@ -243,3 +243,11 @@ significantly more robust than previous client-side Wayland screen locking
...
 to be unlocked.")
     (license license:isc)))
+
+(define-public waybar-experimental
+  (let ((base waybar))
+    (package
+     (inherit base)
+     (name "waybar-experimental")
+     (arguments
+      (append (list #:configure-flags #~(list "-Dexperimental=true")))))))
```
Comment (rakino) [inline rosenthal/packages/wm.scm:253]: As there's only one operand, `append` is actually unnecessary.
```suggestion
      (list #:configure-flags #~(list "-Dexperimental=true"))))))
```

**Comment by threddast** [comment]
Makes sense, I will have a look at the manual

**Comment by threddast** [comment]
I finally found the time to do it: https://debbugs.gnu.org/cgi/bugreport.cgi?bug=61545

**Pull request closed without merging**